packages/desktop/lib/electron/preloads/transak.preload.ts

```diff
@@ -35,7 +35,13 @@ window.addEventListener('DOMContentLoaded', () => {
         iframe.style.border = 'none'
         iframe.style.borderRadius = '16px'
         iframe.allow = 'camera;microphone;payment'
-        iframe.sandbox.add('allow-scripts', 'allow-same-origin')
+        iframe.sandbox.add(
+            'allow-scripts',
+            'allow-forms',
```

Comment:
In which scenario do we need to allow forms? Maybe add a comment if required.

Reply:
iframe will be removed in another PR